app/models/project.rb

```diff
@@ -42,6 +42,8 @@ class Project < ApplicationRecord
   scope :unassigned_to_user, -> { where assigned_to: nil }
   scope :assigned_to_regional_caseworker_team, -> { where(assigned_to_regional_caseworker_team: true) }
 
+  scope :opening_by_month_year, ->(month, year) { where.not(conversion_date: nil).and(where("YEAR(conversion_date) = ?", year)).and(where("MONTH(conversion_date) = ?", month)) }
```

Comment:
Clever use of SQL here!

Reply:
Had to do a bit of googling to get the MSSQL syntax!! `YEAR()` and `MONTH()` are apparently MSSQL specific.